marshmallow/schema.py

```diff
@@ -173,6 +173,16 @@ def _resolve_processors(self):
                 # the processor was a descriptor or something.
                 self.__processors__[tag].append(attr_name)
 
+        # Sort all processor tags by priority, descending
+        for tag, processors in self.__processors__.items():
+            sorted_processors = sorted(processors,
+                                       key=lambda proc: self._get_priority(proc, tag))
+            self.__processors__[tag] = sorted_processors
+
+    def _get_priority(self, attr_name, tag):
+        processor = getattr(self, attr_name)
+        return processor.__marshmallow_kwargs__[tag].get('priority', 0)
```

Comment:
Do you need that default value for priority?

Since a default is set above, this should work:

```py
    return processor.__marshmallow_kwargs__[tag]['priority']
```